src/mixins/date-mixin.ts

```diff
@@ -81,31 +81,20 @@ export default class DateMixin extends Mixins(CommonMixin) {
    * @example "2021-01-01 07:00:00 GMT" -> "Dec 31, 2020"
    * @example "2021-01-01 08:00:00 GMT" -> "Jan 1, 2021"
    */
-  dateToPacificDate (date: Date, long = false): string {
+  dateToPacificDate (date: Date, longMonth = false, showWeekday = false): string {
```

Comment:
I separated out long month and whether to show the weekday. This makes this method compatible with our other projects.
Eg1, normal usage: `this.dateToPacificDate(date)`
Eg2, called from dateToPacificDateTime:  `this.dateToPacificDate(date, true)`
Eg3, special case where you also want the weekday:  `this.dateToPacificDate(date, true, true)`